Resources/public/js/views/fields/ez-dateandtime-view.js

```diff
@@ -42,6 +42,7 @@ YUI.add('ez-dateandtime-view', function (Y) {
          * Formats the date part of the date object according to the locale
          * settings of the browser
          *
+         * @deprecated Since 1.7.1
```

Comment:
you should add the console.log calls in the method ([example](https://github.com/ezsystems/PlatformUIBundle/blob/bd887d42fe34896fde6ba8b04508d5e046995860/Resources/public/js/views/ez-languageswitcherview.js#L136)) so that the notice is displayed in the console.